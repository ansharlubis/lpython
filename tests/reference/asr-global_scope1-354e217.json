{
    "basename": "asr-global_scope1-354e217",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/global_scope1.py",
    "infile_hash": "e7b33a2680429ee47d15af5af7c31ac24cc802d18d5566b568fc8ef4",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-global_scope1-354e217.stdout",
    "stdout_hash": "304d1f02a91400e074440ee6740380d3624f753be56bb6c0ab441a24",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}