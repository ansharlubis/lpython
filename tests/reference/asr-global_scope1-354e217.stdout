([1m[35mTranslationUnit[39m[0m 
   ([33mSymbolTable[39m 
      1 
      {
         i: 
            ([1m[35mVariable[39m[0m 
               1 
               i [1m[32m
               Local[39m[0m () () [1m[32m
               Default[39m[0m 
               ([1m[35mInteger[39m[0m 4 []) [1m[32m
               Source[39m[0m [1m[32m
               Public[39m[0m [1m[32m
               Required[39m[0m .false.)
      }) 
   [
   ([1m[35m=[39m[0m 
      ([1m[35mVar[39m[0m 1 [33mi[39m) ([1m[35mIntegerConstant[39m[0m [36m5[39m 
      ([1m[35mInteger[39m[0m 4 [])) ())])
(TranslationUnit (SymbolTable 1 {_lpython_main_program: (Subroutine (SymbolTable 3 {}) _lpython_main_program [] [(= (Var 1 i) (IntegerConstant 5 (Integer 4 [])) ())] Source Public Implementation () .false. .false.), i: (Variable 1 i Local () () Default (Integer 4 []) Source Public Required .false.), main_program: (Program (SymbolTable 2 {}) main_program [] [(SubroutineCall 1 _lpython_main_program () [] ())])}) [])
